eftcamb/11_EFTCAMB_stability.f90

```diff
@@ -33,211 +33,119 @@ module EFTCAMB_stability
     use EFTCAMB_abstract_model_full
     use EFTCAMB_abstract_model_designer
     use EFTDef
+    use EFTCAMB_cache
 
     implicit none
 
 contains
 
     ! ---------------------------------------------------------------------------------------------
     !> Function that computes if the stability requirements are fullfilled in a given time.
-    function EFTStabilityComputation(a)
+    function EFTStabilityComputation( self, a, eft_par_cache, eft_cache )
```

Comment:
We need this otherwise compilation does not work.
We have to give eft_par_cache and eft_cache as external parameters.